classes/Rest/Controllers/AppKernelControllerProvider.php

```diff
@@ -1602,10 +1602,6 @@ public function getRawPerformanceMap(Request $request, Application $app)
         // utilize this endpoint. Note, we do not utilize the `requirements` parameter of the above
         // `authorize` call because it utilizes `XDUser::hasAcls` which only checks if the user has
         // *all* of the supplied acls, not any of the supplied acls.
-        if ( ! ( $user->hasAcl(ROLE_ID_CENTER_DIRECTOR) ||  $user->hasAcl(ROLE_ID_CENTER_STAFF) ) ) {
```

Comment:
Please update the function comment and inline comments as appropriate.

Reply:
Thanks! updated in latest commit.